plugin/storage/es/spanstore/dbmodel/from_domain_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dbmodel
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+const NumberOfFixtures = 1
+
+func TestFromDomainEmbedProcess(t *testing.T) {
+	for i := 1; i <= NumberOfFixtures; i++ {
+		domainStr, jsonStr := loadFixtures(t, i)
+
+		var span model.Span
+		require.NoError(t, jsonpb.Unmarshal(bytes.NewReader(domainStr), &span))
+		converter := NewFromDomain(false, nil, ":")
+		embeddedSpan := converter.FromDomainEmbedProcess(&span)
+
+		var expectedSpan Span
+		require.NoError(t, json.Unmarshal(jsonStr, &expectedSpan))
+
+		testJSONEncoding(t, i, jsonStr, embeddedSpan)
```

Comment:
Could you wrap each test in a t.Run() with an appropriate name? Like, "TestFromDomainEmbedProcess#fixture%d"